docs/asyncapi.yaml

```diff
@@ -469,10 +404,10 @@ components:
       type: string
       format: DD-MM-YYYY
       description: Date SITI Agri exported payment file
-      example: "2022-06-28"
+      example: '2022-06-28'
     Checkpoint:
       type: string
-      description: Service which raised the event, differs per environment
+      description: 'Service which raised the event, differs per environment'
```

Comment:
I don't think the single quotes are needed here

Reply:
Agreed, another one I didn't change (I promise!).  Removed quotes